src/libs/actions/IOU.js

```diff
@@ -13,7 +16,117 @@ function getPreferredCurrency() {
     }, 1600);
 }
 
-// Re-enable the prefer-default-export lint when additional functions are added
+/**
+ * Creates IOUSplit Transaction
+ * @param {Object} parameters
+ * @param {String} parameters.amount
+ * @param {String} parameters.comment
+ * @param {String} parameters.currency
+ * @param {String} parameters.debtorEmail
+ */
+function createIOUTransaction({
+    comment, amount, currency, debtorEmail,
+}) {
+    Onyx.merge(ONYXKEYS.IOU, {loading: true, creatingIOUTransaction: true, error: false});
+    let iouReportID = '';
+    API.CreateIOUTransaction({
+        comment,
+        amount,
+        currency,
+        debtorEmail,
+    })
+        .then((data) => {
+            iouReportID = data.reportID;
+            return iouReportID;
+        })
+        .then(reportID => API.Get({
+            returnValueList: 'reportStuff',
+            reportIDList: reportID,
+            shouldLoadOptionalKeys: true,
+            includePinnedReports: true,
+        }))
+        .then((response) => {
+            if (response.jsonCode !== 200) {
+                throw new Error(response.message);
+            }
+
+            const iouReportData = response.reports[iouReportID];
+            if (!iouReportData) {
+                throw new Error(`No iouReportData found for reportID ${iouReportID}`);
+            }
+
+            Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_IOUS}${iouReportID}`,
+                getSimplifiedIOUReport(iouReportData));
+            Onyx.merge(ONYXKEYS.IOU, {loading: false, creatingIOUTransaction: false, error: false});
+        })
+        .catch((error) => {
+            Onyx.merge(ONYXKEYS.IOU, {loading: false, creatingIOUTransaction: false, error: true});
+            throw new Error(`[Report] Failed to populate IOU Collection: ${error.message}`);
+        });
+}
+
+/**
+ * Creates IOUSplit Transaction
+ * @param {Object} parameters
+ * @param {Array} parameters.participants
+ * @param {Array} parameters.splits
+ * @param {String} parameters.comment
+ * @param {String} parameters.amount
+ * @param {String} parameters.currency
+ */
+function createIOUSplit({
+    participants,
+    comment,
+    amount,
+    currency,
+    splits,
+}) {
```

Comment:
Although the participants and splits params are different, there is quite a bit of shared logic. 

Can you refactor the logic that starts with `reportStuff` to a new function. Then `createIOUSplit` and `createIOUTransaction` could both call it.